pkg/tfvar/tfvar.go

```diff
@@ -110,7 +112,62 @@ func WriteAsTFVars(w io.Writer, vars []Variable) error {
 	_, err := f.WriteTo(w)
 	return errors.Wrap(err, "tfvar: failed to write as tfvars")
 }
+func WriteAsWorkspacePayload(w io.Writer, vars []Variable) error {
+	var payload error
+	// type SimpleJSONValue struct {
+	// 	cty.Value
+	// }
 
+	for _, v := range vars {
+		val := convertNull(v.Value)
+
+		t := hclwrite.TokensForValue(val)
+		t = oneliner(t)
+		b := hclwrite.Format(t.Bytes())
+		b = bytes.TrimPrefix(b, []byte(`"`))
+		b = bytes.TrimSuffix(b, []byte(`"`))
+		b = bytes.ReplaceAll(b, []byte(`"`), []byte(`'`))
+		data := fmt.Sprintf(`{
+			"data": {
+				"type": "vars",
+				"attributes": {
+					"key":         "%s",
+					"value":       "%s",
+					"description": "%s",
+					"category":    "%s",
+					"hcl":         %s,
+					"sensitive":   %s
+				}
+			}
+		}
```

Comment:
It seems that we are building a JSON payload here. Do you think we can improve this part to use a struct with json.Marshal?

Reply:
Probably but I honestly could not get it to work using a structure and `json.Marshal`, this is most likely my lack of experience with GO and the cty.value conversion , the above I got working and it made more sense to me